ci/azure-pipelines.yml

```diff
@@ -114,6 +114,26 @@ stages:
             displayName: Build on Ubuntu
             condition: contains(variables['build_container'], 'ubuntu')
 
+          - bash: |
+              set -eEx
+              ./autogen.sh
+              export CFLAGS='-Werror -Wall'
+              export KFLAGS='-Werror'
```

Comment:
Can you please remove KFLAGS for both?

Reply:
also since it rebuilds existing tree I guess we'd need to git clean. currently CI did not rebuild it.

Reply:
It is fixed now. I somehow missed it. Thank you for spotting that.

Reply:
Fixed. Didn't realize that each one was done right after another in place (new to azure pipelines).

Reply:
I changed the line to be `make -i distclean || true` so that `make` ignores errors and that the exit code is always using `true` instead of `:` (makes it more clear)

Reply:
Now all CI checks pass

Reply:
i see ;) eventually maybe it's simpler to use like `test -f Makefile && make -i distclean || true` before autgen.sh

Reply:
I am not sure which is better. It probably makes little difference